xml/System.Windows.Forms/AccessibleEvents.xml

```diff
@@ -25,7 +25,7 @@
   
  This enumeration is used by <xref:System.Windows.Forms.AccessibleObject> and <xref:System.Windows.Forms.Control>.  
   
- For more information about the accessibility application, search for "Microsoft Active Accessibility" in the MSDN Library.  
+ For more information, see [Microsoft Active Accessibility](/windows/win32/winauto/microsoft-active-accessibility).
   
    
```

Comment:
The build warning is not related to your changes but probably should be addressed at some point. The remarks from fields should be moved here since they're not displayed on the page. Ron has done some similar changes in the past to address this problem for some APIs, but we probably missed a few.

Reply:
Thanks. I have an issue assigned to myself to move the remarks into the summaries - https://github.com/dotnet/dotnet-api-docs/issues/5220. The build warning was added at my request.